### PR TITLE
Immediately validate x-frame-options when opening saved Settings

### PR DESCRIPTION
**Pull request by stulees**
- Added `dist/` directory to gitignore
- Updated `circle.yml` to adhere to latest configurations
- Updated bower and npm to latest rise releases
- Configured widget for ESLinting and fixed all lint issues
- Fixed issue #70

### CODE REVIEW

**Comment by stulees** [comment]
@tejohnso and/or @settinghead Please review, thanks. Last commit fixes issue #70

File: gulpfile.js [hunk 1]
```diff
@@ -177,6 +177,10 @@
     });
   });
 
+  gulp.task("build-dev", (cb) => {
```
Comment (settinghead) [inline gulpfile.js]: I think these tasks expect a return value in order to be chained together. In this case `return runSequence(...)` should do.

Comment (stulees) [inline gulpfile.js]: @settinghead "build-dev" is just a sequence of tasks in that order, the task itself doesn't need to return a value. It does ensure to execute a provided callback function when the sequence has completed. "build-dev" is the exact same as "build" task just without the "bower-update" task.

Comment (settinghead) [inline gulpfile.js]: @stulees oh you are right. Never mind what I said.

**Comment by settinghead** [comment]
:+1:

**Pull request merged**